modules/standalone-georepl-site-removal.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: PROCEDURE
+[id="standalone-georepl-site-removal"]
+= Removing a geo-replicated site from your standalone {productname} deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to execute a shell inside of the database pod:
```

Comment:
It's not correct, the command to remove site should be executed inside Quay APP POD, not quay database pod